### PR TITLE
Replaced CompositeInterceptor with AuthorizationAndInternalUserInterc…

### PR DESCRIPTION
**Pull request by krishna-patel-ch**
__Short description outlining key changes/additions__

Replaced CompositeInterceptor with AuthorizationAndInternalUserInterceptors

__JIRA Ticket Number__

### Type of change

IDVA6-1348

* [x] Bug fix
* [x] New feature
* [x] Breaking change
* [x] Infrastructure change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Review by kkonuganti-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/AuthorizationAndInternalUserInterceptors.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    private final InternalUserInterceptor internalUserInterceptor;
+
+    public AuthorizationAndInternalUserInterceptors( final UsersService usersService ) {
+        super( usersService );
+        this.internalUserInterceptor = new InternalUserInterceptor( StaticPropertyUtil.APPLICATION_NAMESPACE );
+    }
+
+    @Override
+    public boolean preHandle( HttpServletRequest request, HttpServletResponse response, Object handler ) {
```
Comment (tferns-ch) [inline src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/AuthorizationAndInternalUserInterceptors.java:23]: Why the change?  I preferred the other approach you had which was preferring composition (or aggregation) over inheritance.  Not like it's a big deal, this one is similar to the initial example I sent you this morning, but with a few minor differences, just curious on the decision to change is all.

**Review by tferns-ch — APPROVED** [review]
Looks good to me, just a question
